paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodDefinition.kt

```diff
@@ -18,7 +18,7 @@ internal interface PaymentMethodDefinition {
      * Or setOf(AddPaymentMethodRequirement.MerchantSupportsDelayedPaymentMethods) if the payment method requires the
      * merchant to provide a PaymentSheet.Configuration with delayed payment methods enabled.
      */
-    fun addRequirement(hasIntentToSetup: Boolean): Set<AddPaymentMethodRequirement>
+    fun addRequirements(hasIntentToSetup: Boolean): Set<AddPaymentMethodRequirement>
```

Comment:
The naming is a bit odd, as it sounds like you’re _adding_ requirements.

Reply:
Discussed offline, and came up with `requirementsToBeUsedAsNewPaymentMethod`